shell/src/main/java/alluxio/shell/command/ChownCommand.java

```diff
@@ -76,18 +83,46 @@ private void chown(AlluxioURI path, String owner, boolean recursive)
     System.out.println("Changed owner of " + path + " to " + owner);
   }
 
+  /**
+   * Changes the owner and group for the directory or file with the path specified in args.
```

Comment:
"path" means file or directory, so we can just say "Changes the owner and group for the specified path"